test/compile.jl

```diff
@@ -220,7 +220,8 @@ try
             Dict(s => Base.module_uuid(Base.root_module(s)) for s in
                 [:Base64, :CRC32c, :Dates, :DelimitedFiles, :Distributed, :FileWatching,
                  :Future, :IterativeEigensolvers, :Libdl, :LinearAlgebra, :Logging, :Mmap, :Printf,
-                 :Profile, :Random, :Serialization, :SharedArrays, :SparseArrays, :SuiteSparse, :Test, :Unicode]))
+                 :Profile, :Random, :Serialization, :SharedArrays, :SparseArrays, :SuiteSparse, :Test,
+                 :Unicode, :REPL]))
```

Comment:
These module names are sorted alphabetically, it may not be so useful, but why not keep the order :)

Reply:
I wanted to keep the precompile statements as close to where they originally were.